airflow/serialization/serializers/timezone.py

```diff
@@ -29,10 +30,15 @@
 serializers = [
     "pendulum.tz.timezone.FixedTimezone",
     "pendulum.tz.timezone.Timezone",
-    "zoneinfo.ZoneInfo",
-    "backports.zoneinfo.ZoneInfo",
 ]
 
+PY39 = sys.version_info >= (3, 9)
```

Comment:
PY39 is defined at airflow/__init__.py so we should probably import from there?:
`from airflow import PY39`

Reply:
It was argued not to do that to prevent potential circular imports